decidim-core/spec/system/session_timeout_spec.rb

```diff
@@ -10,16 +10,16 @@
 
   context "when session is about to timeout" do
     before do
-      Devise.timeout_in = 2.minutes
-      Decidim.config.session_timeouter_interval = 1000
+      Decidim.expire_session_after = 2.minutes
+      Decidim.session_timeout_interval = 1.second
```

Comment:
These have to be changed accordingly to #7525 after that PR is merged.

The method right now changes the global configuration for the whole test app which is problematic when this test is run along with other tests (as the CI does).